.github/workflows/test.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: test
+on: pull_request
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4.1.1
+      - uses: actions/setup-go@v5.0.0
+        with:
+          go-version: '1.21.5'
+      - run: make lint
+      - run: make vet
```

Comment:
There's no such target: vet, but `make lint` should do the vet job too

Reply:
Oh, sorry. This is a typo of `test`. I'll fix.

Reply:
Fixed. 87c20151e354d5612de0c573ad77d3fff5c4d15b